src/SFA.DAS.ApplyService.Data/ApplyRepository.cs

```diff
@@ -1226,5 +1226,238 @@ public async Task<bool> UpdateApplyData(Guid applicationId, ApplyData applyData,
+        {
+            using (var connection = new SqlConnection(_config.SqlConnectionString))
+            {
+                var pageReviewOutcomeResults = await connection.QueryAsync<PageReviewOutcome>(@"IF (@assessorType = 1)
+	                                                                BEGIN
+		                                                                SELECT [ApplicationId]
+			                                                                  ,[SequenceNumber]
+			                                                                  ,[SectionNumber]
+			                                                                  ,[PageId]
+			                                                                  ,@assessorType AS AssessorType
+			                                                                  ,[Assessor1UserId] AS UserId
+			                                                                  ,[Assessor1ReviewStatus] AS [Status]
+			                                                                  ,[Assessor1ReviewComment] AS Comment
+		                                                                  FROM [dbo].[AssessorPageReviewOutcome]
+		                                                                  WHERE [ApplicationId] = @applicationId AND
+				                                                                [Assessor1UserId] = @userId                                                        
+	                                                                END
+                                                                IF (@assessorType = 2)
+	                                                                BEGIN
+		                                                                SELECT [ApplicationId]
+			                                                                  ,[SequenceNumber]
+			                                                                  ,[SectionNumber]
+			                                                                  ,[PageId]
+			                                                                  ,@assessorType AS AssessorType
+			                                                                  ,[Assessor2UserId] AS UserId
+			                                                                  ,[Assessor2ReviewStatus] AS [Status]
+			                                                                  ,[Assessor2ReviewComment] AS Comment
+		                                                                  FROM [dbo].[AssessorPageReviewOutcome]
+		                                                                  WHERE [ApplicationId] = @applicationId AND
+				                                                                [Assessor2UserId] = @userId                      
+	                                                                END",
+                    new { applicationId, assessorType, userId });
+
+                return pageReviewOutcomeResults.ToList();
+            }
+        }
```

Comment:
ApplyRepository is getting massive now. I'll raise a tech debt ticket to look at splitting it up

Reply:
Yes, It's about time to do that

Reply:
Why don't we do it as part of this ticket then?  I've already created an AssessorRepository so maybe these methods could go on there?

Reply:
Methods moved to AssessorRepository => PR 549